less/v2/chat.less

```diff
@@ -191,7 +191,7 @@
         gap: 1px;
 
         .roll {
-          position: unset;
+          position: relative;
```

Comment:
Since core already defines relative positioning here, we can just remove this definition and rely on inheritance.

```suggestion
```